### PR TITLE
Use sandbox + srcdoc and postMessage for updating the url.

### LINKED ISSUE #99: Should the iframe point to a different domain?
Firstly, the app is really cool. Hats off!!

In order to prevent XSS attacks and other security issues, should the iframe in ember-twiddle point to a different domain? Similar to JSFiddle. Currently, we can access the parent window from the iframe code and can access the browser cookies as well.  

Correct me if my understanding is wrong.

### PR DESCRIPTION
**Pull request by joostdevries**
Fixes #99. THIS WIL BREAK IE.

- [x] Implement sandbox
- [x] Fix broken tests
- [ ] Add test for sandbox
- [x] Add blocking message for IE / unsupported browser users

### CODE REVIEW

**Comment by joostdevries** [comment]
@Gaurav0 this is quite tricky since the tests need access to the iframe. I could disable the sandbox during testing but that would kind of defeat the point of the test right?

**Comment by Gaurav0** [comment]
The more I think about it the more I think we need to go with the mutlple domains.

**Comment by joostdevries** [comment]
You will have the same problem.

**Comment by Gaurav0** [comment]
I guess I could accept a toggle based on testing as long as there was no use of window.parent from the iframe.

**Comment by Gaurav0** [comment]
Can you enforce that with a window.parent = window?

File: app/gist/controller.js [hunk 1]
```diff
@@ -50,6 +49,7 @@ export default Em.Controller.extend({
       this.set('buildOutput', buildOutput);
     })
     .catch(errors => {
+      console.log(errors);
```
Comment (Gaurav0) [inline app/gist/controller.js]: Do you need this still?

Comment (joostdevries) [inline app/gist/controller.js]: sorry no.. will remove.

**Comment by joostdevries** [comment]
We need a nice error message for IE users but the benefit of srcdoc is that it will just not work (so we don't have to block manually).

**Comment by joostdevries** [comment]
Also https://github.com/jugglinmike/srcdoc-polyfill

**Comment by Gaurav0** [comment]
Is the polyfill secure?

**Comment by joostdevries** [comment]
would like to merge as-is and release, if someone could signoff that'd be great.

**Comment by joostdevries** [comment]
image

**Comment by stefanpenner** [comment]
Polyfil looks safe and legit. some browser do have a uri limit (bug) but I think it's only safari that hits the issue with relatively low size. 4k. And safari supports real srcdoc, so all seems well based on my current recollection 

+1

**Comment by stefanpenner** [comment]
This looks good as is. The poly can come now or later.

**Pull request merged**